### ISSUE TITLE
When execute .JAR once boot up, why does the program run once to make RESET active?

### ISSUE THREAD

**Issue by Jir8taiwan** [panovvv/hardware-watchdog-arduino#2]
Hi, the Builder whom may concern,

I try to use your share to apply in my win10 computer, but cannot run correctly.
I have compiler the .JAR and upload watchdog program to Arduino without problem.
The only problem is not understood now.
I create an execute file in windows startup folder to make communicate with arduino.
However, I detect the .JAR original compiled file which is strange.
When it is the first run once, it will make relay circuit to run once and go starting communication with arduino.
This kind of malfunction will make directly reboot due to relay of normal close pin which is back again at the moment.
In other words, computer is always reboot with act repeats, when it is just booting to desktop and starting to do monitoring.
I am not sure where is wrong, but would you please help me to figure out possible root cause?
I upload a video in private youtube link for problem reference as here:
https://youtu.be/nJmoETYxx9Q

Thanks for attention.
